GitUI/UserControls/RevisionGrid.cs

```diff
@@ -1693,12 +1693,12 @@ public struct DrawRefArgs
                         int gravatarTop = e.CellBounds.Top + textHeight + 6;
                         int gravatarLeft = e.CellBounds.Left + baseOffset + 2;
 
-                        var gravatar = _avatarCache.GetImage($"{revision.AuthorEmail}.png", null);
+                        var gravatar = _gravatarService.GetImage(revision.AuthorEmail, null);
```

Comment:
There is sth wrong here, we should not know at this point how AuthorEmail translates into the `$"{revision.AuthorEmail}.png"` - we should work against _gravatarService public interface.